conf/broker.conf

```diff
@@ -930,6 +930,22 @@ bookkeeperExplicitLacIntervalInMills=0
 # Expose bookkeeper client managed ledger stats to prometheus. default is false
 # bookkeeperClientExposeStatsToPrometheus=false
 
+# Timeout controlling wait on request send in case of unresponsive bookie(s).
+#    Timeout value:
+#         - negative value disables the feature
+#         - positive to allow wait up to specified number of milliseconds
+#         - 0 to allow request to fail immediately.
+#    Default is -1 (disabled)
+#########
+#  # Should be used in combination with the bookkeeper server-side
+#  #           backpressure configuration, such as
+#  #   maxAddsInProgressLimit = (positive value)
+#  #   maxReadsInProgressLimit = (positive value)
+#  #   closeChannelOnResponseTimeout = true
+#  #   waitTimeoutOnResponseBackpressureMs = (positive value)
+#########
+# bookkeeper_waitTimeoutOnBackpressureMillis = -1
```

Comment:
Is there a reason we should keep this config commented out? I think it would be helpful to make this a first class member of this file so that users don't need to add the `PULSAR_PREFIX_` to the env var.